tools/marvin/setup.py

```diff
@@ -27,7 +27,7 @@
         raise RuntimeError("python setuptools is required to build Marvin")
 
 
-VERSION = "4.20.0.0-SNAPSHOT"
+VERSION = "4.20.0.0"
```

Comment:
```suggestion
VERSION = "4.20.0.0-SNAPSHOT"
```

Reply:
I think this change should not be here, but also this is easy to miss. should it be removed?